Sources/Rendering/Core/TextProperty/index.js

```diff
@@ -0,0 +1,64 @@
+import macro from 'vtk.js/Sources/macros';
+import vtkProperty2D from 'vtk.js/Sources/Rendering/Core/Property2D';
+
+// ----------------------------------------------------------------------------
+// vtkTextProperty methods
+// ----------------------------------------------------------------------------
+
+function vtkTextProperty(publicAPI, model) {
+  // Set our className
+  model.classHierarchy.push('vtkTextProperty');
+}
+
+// ----------------------------------------------------------------------------
+// Object factory
+// ----------------------------------------------------------------------------
+
+const DEFAULT_VALUES = {
+  fontFamily: 'Arial',
+  fontColor: [0, 0, 0],
+  fontStyle: 'normal',
+  backgroundColor: [1, 1, 1],
+  fontSizeScale: (resolution) => resolution / 1.8,
```

Comment:
please add a comment that explains why 1.8

Reply:
I have added a comment, FYI i am using the same method used in vtkAnnotatedCubeActor

https://github.com/Kitware/vtk-js/blob/85784474c7cc4cd60f8ea0636b3d10dd367343e3/Sources/Rendering/Core/AnnotatedCubeActor/index.js#L151